test/integration/kubernetes_deploy_test.rb

```diff
@@ -1006,4 +1006,21 @@ def test_raise_on_yaml_missing_kind
       "  datapoint: value1"
     ], in_order: true)
   end
+
+  def test_hpa_can_be_successful
+    skip if KUBE_SERVER_VERSION < Gem::Version.new('1.8.0')
+    assert_deploy_success(deploy_fixtures("hpa"))
```

Comment:
Please try to include at least one relevant logging assertion with your tests. The logs are our UI, so we should always check (`PRINT_LOGS=1`) what they look like and assert that we haven't broken them. Typically I'll assert on what gets printed about the resource in the summary